aiosignal/__init__.py

```diff
@@ -16,21 +29,26 @@ class Signal(FrozenList):
 
     __slots__ = ("_owner",)
 
-    def __init__(self, owner):
+    def __init__(self, owner:_O):
```

Comment:
The owner parameter doesn't seem to have any actual use, so I don't see any reason to track it with a TypeVar:
```suggestion
    def __init__(self, owner: object):
```

Reply:
@Dreamsorcerer Makes sense. I just didn't know if users wanted to remember what objects were being carried around or not.